src/cast_core/include/cast_core/actors/ExternalScriptRunner.hpp

```diff
@@ -52,6 +52,7 @@ class ExternalScriptRunner : public Actor {
     struct PhaseConfig;
     PhaseLoop<PhaseConfig> _loop;
     std::string _command;
+    std::unordered_map<std::string, std::string> _environmentVariables;
```

Comment:
Is this new and/or needed? I don't see it referenced in the other changes.

Reply:
This is supposed to be from Brad's branch which I merge from. He might have removed it later. I am removing it from here.